modules/enabling-external-ip-collection-secured-clusters.adoc

```diff
@@ -27,9 +27,11 @@ data:
   runtime_config.yaml: | <1>
     networking:
       externalIps:
-        enable: true
+        enabled: ENABLED <2>
 ----
 <1> {product-title-short} mounts this file at `/etc/stackrox/runtime_config.yaml`.
+<2> Please note that `networking.externalIps.enable` has been changed to `networking.externalIps.enabled` in {product-title-short} 4.7. It is now an enum, and can be set to `ENABLED` or `DISABLED`.
```

Comment:
```suggestion
<2> `networking.externalIps.enable` has been changed to `networking.externalIps.enabled` in {product-title-short} 4.7. It is now an enum, and can be set to `ENABLED` or `DISABLED`.
```
(we don't use "please" in technical documentation and note is reserved for things set off in the "note" format)

Reply:
Done